packages/topbar/src/FzTopbar.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div :class="containerClass">
+    <span> Testo qui </span>
```

Comment:
You should probably use `<slot></slot>` here and not this static text

Reply:
oh yeah my bad, good catch

Reply:
done in 13063ef